### PR TITLE
Add effects to RGB LED

### PR DESCRIPTION
**Pull request by btli**
Home Alarm system is integrated with HA. This is a small addition to give AIOsense the ability to use a few LED effects and provide a visual indicator as these are placed in every room of the house. Also, made the strobing configurable since the default timings are a bit too calm for my preferences.

### CODE REVIEW

**Comment by Schluggi** [comment]
Looks nice. I'll check this soon :)

File: esphome/packages/sensors/rgb_led.yaml [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # https://esphome.io/components/light/neopixelbus.html
+substitutions:
+  strobe_on: 50ms
```
Comment (Schluggi) [inline esphome/packages/sensors/rgb_led.yaml:3]: Please prefix your substitutions with `rgb_led_effects_` and add them also as a comment to the `esphome.yaml`.

**Review by Schluggi — CHANGES_REQUESTED** [review]

**Review by Schluggi — APPROVED** [review]

**Comment by Schluggi** [comment]
Please fix the ynal linting. You can ignore all "line too long" errors.

**Comment by btli** [comment]
Don't have a D1 to test, but tried compiling. LED doesn't support pulse effect, so removed that. Should be passing tests now.

**Pull request merged**